client/post-editor/editor-title/index.jsx

```diff
@@ -45,37 +41,40 @@ const EditorTitle = React.createClass( {
 		};
 	},
 
-	componentWillReceiveProps( nextProps ) {
+	componentDidUpdate( prevProps ) {
 		if ( isMobile() ) {
 			return;
 		}
 
 		// If next post is new, or the next site is different, focus title
-		if ( nextProps.isNew && ! this.props.isNew ||
-			( nextProps.isNew && ( this.props.site && nextProps.site ) && ( this.props.site.ID !== nextProps.site.ID ) )
-		) {
-			this.setState( {
-				isFocused: true
-			}, () => {
-				this.refs.titleInput.focus();
-			} );
+		const { isNew, site } = this.props;
+		if ( ( isNew && ! prevProps.isNew ) ||
+				( isNew && get( prevProps.site, 'ID' ) !== get( site, 'ID' ) ) ) {
```

Comment:
>I think this indentation style is clearer here:

I go back and forth myself, but FWIW this is the documented style:

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#multi-line-statements